deploy/helm/sumologic/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  repository: sumologic/kubernetes-fluentd
+  tag: 0.0.0
+  pullPolicy: Always
```

Comment:
When we do make this officially released, this should default to IfNotPresent, not always.